data/14.0.0.xml

```diff
@@ -5,14 +5,14 @@
     <changelog href="https://nextcloud.com/changelog/#14-0-0" />
     <whatsNew lang="en">
         <regular>
-            <item>Verify the identity of a share recipient in a video call; and 2FA with Signal &amp; Telegram</item>
-            <item>Add a note to shares, see pending &amp; deleted shares and search for files by content of comments</item>
-            <item>Accessibility improvements including dark &amp; high contrast themes and a dyslectic font</item>
+            <item>Verify a share recipient in a video call; 2FA with Signal &amp; Telegram</item>
```

Comment:
If any, I would mention that 2FA via **SMS** is now supported. In contrast to Signal and Telegram, this has been tested more thoroughly. Still, the whole app must not be considered production ready. It's an early beta at best.

Reply:
On a general note, I always feel way more comfortable to only announce features that are done and stable.
Otherwise it leads to more pressure to the devs (and we all have plenty of things to do).

Reply:
@jospoortvliet that feedback is still there (and I still agree, but whatever i prefer a decision and see PD as owner of that).